src/components/DateRangePickerInput.jsx

```diff
@@ -145,13 +144,16 @@ export default class DateRangePickerInput extends React.Component {
         })}
       >
         {(showDefaultInputIcon || customInputIcon !== null) &&
-          <span
+          <button
```

Comment:
Yes. I figured we'd do some of the small aria-labeling/add those phrases unrelated to keyboard nav here.